### PR TITLE
#1 - Roles

### PR DESCRIPTION
**Pull request by miromichalicka**
Creating pull request for feature Roles. There are also all permission for all roles.

### CODE REVIEW

File: docroot/profiles/campsite/modules/features/roles/roles.features.user_permission.inc [hunk 1]
```diff
@@ -0,0 +1,715 @@
+<?php
```
Comment (Sweetchuck) [inline docroot/profiles/campsite/modules/features/roles/roles.features.user_permission.inc:1]: The file path should be `docroot/profiles/campsite/modules/custom/**`

**Comment by csakiistvan** [comment]
If I see correctly, this PR set permissions, but the #29 issue doesnt include permissions, only roles. And @Sweetchuck remove some module from profile info dependencies, se if we not use block module, we don't need set its permissions, now

**Comment by Sweetchuck** [comment]
yesterday night I played with the CKeditor and I needed some role as well.

This feature contains user roles, text formats and CKEditor profiles.
https://github.com/Sweetchuck/campsite-2015/tree/i12-ckeditor/docroot/profiles/campsite/modules/custom/campsite_core

**Comment by miromichalicka** [comment]
Ok, so then I will just extend those campsite_core feature, I don't find very clever to have permissions and roles separated in many modules

**Pull request closed without merging**